packages/webpack-config/src/helpers/getFileExtension/getFileExtension.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Get the file extension from a filepath or filename.
+ *
+ * @param {string} file - filename or filepath
+ * @returns {string} file extension
+ */
+function getFileExtension(file) {
+  return file.split(".").reverse()[0];
```

Comment:
maybe you could, handle the case if there is no "." in filename/filepath

Reply:
we should also have a list of possible filetypes (js, jsx, ts, tsx) and throw a warning/error if non of them fits

Reply:
this is not an issue since JS handles that gracefully.